src/data_management/wms_layers/WmsLayerForm.js

```diff
@@ -102,6 +102,16 @@ class WmsLayerFormModel extends Component {
     }
   }
 
+  optionsValidator = (value) => {
```

Comment:
I think the optionsValidator should also accept null value or empty string as this field is not mandatory to fill in for the user. What do you think? For now, if I don't want to configure this, I need to provide an empty json object {} or click on the X icon on the right to move on.